### ISSUE TITLE
[escaping-literal-quotes-in-strings] Still not working 

### ISSUE THREAD

**Issue by Elijah-Sonnier** [FreeCodeCamp/FreeCodeCamp#10620]
Challenge [escaping-literal-quotes-in-strings](https://www.freecodecamp.com/en/challenges/basic-javascript/escaping-literal-quotes-in-strings) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

```js


var myStr = "I am a \"double quoted\" string inside \"double quotes\"."; // Change this line

![escaping literal quotes](https://cloud.githubusercontent.com/assets/21290175/18400642/cbfdd8dc-76a4-11e6-9b20-cd3b661daf4a.jpg)

It passes the _Variable myStr should contain the string: I am a "double quoted" string inside "double quotes".__
But fails the _You should use two double quotes (") and four escaped double quotes (\")._
```

**Comment by johnanthony797** [comment]
+1

**Comment by marcandre311** [comment]
I have the same issue

**Comment by dhcodes** [comment]
Closed as duplicate of #10258. Thanks for submitting!

**Comment by lind10ra** [comment]
+1
